core/templates/dev/head/exploration_editor/exploration_editor.html

```diff
@@ -572,7 +572,7 @@
     {{ include_js_file('expressions/expressionInterpolationService.js') }}
   </script>
 
+  <script src="{{ASSET_DIR_PREFIX}}/extensions/interactions/baseValidator.js"></script>
```

Comment:
We haven't defined `cache_slug` as a jinja variable. Instead we have `ASSET_DIR_PREFIX`. Do you think using it for extensions is confusing ?